src/pages/BlogPostCollectionPage.tsx

```diff
@@ -52,9 +52,9 @@ const StyledAbstract = styled.div`
 `
 
 const BlogPostCollectionPage: React.VFC = () => {
-  const [tags] = useQueryParam('tags', StringParam)
+  const [categories] = useQueryParam('categories', StringParam)
```

Comment:
Do not need to change this one. we should support two kinds of query params: tags and categories
not change all the tags to categories